### PR TITLE
Create To-do App

### PR DESCRIPTION
**Pull request by PinkMoon25**
Created To-do App, developed with React Class Components.

- Used class components for the to-do app and functional components for page components.
- Implemented event handling and lifecycle methods.
- Added data persisting in local storage.
- Implemented **add**, **edit**, **delete** and **clear all completed** functionality.
- Used react routing to navigate to different sections of the app.
- Used react icons for submit and delete actions in the app.
- Used basic react styling to style the app.

### CODE REVIEW

**Review by Owusu-Desmond — CHANGES_REQUESTED** [review]
Hi @PinkMoon25 ,

Good job so far! 👍 
Nice touch implementing all the optional requirements for this project ( i.e saving your todos in the localStorage). There are suggestions that you need to work on to make your project better but you are almost there!

almost-there-almost


## To highlight :champagne: 
- All linters are passing :heavy_check_mark: 
- You  are following GitHub actions :heavy_check_mark: 
- Good commit history :heavy_check_mark: 
-  Good PR :heavy_check_mark: 
- Good project structure. :heavy_check_mark: 

## Required Changes ♻️

- [x] Please, kindly change all your `button`  elements cursor to `pointer` to make it more interactive.

- [x] Please check the comments of this review and there are additional suggestions you need to work on to improve your work.

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._


Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear and do not forget to tag me.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Comment by PinkMoon25** [comment]
Thank you @Owusu-Desmond for the review :+1:

File: src/components/TodoContainer.js [hunk 1]
```diff
@@ -0,0 +1,107 @@
...
+    }));
+  }
+
+  addTasks(task) {
+    const { todos } = this.state;
+    const newTask = {
+      id: Math.floor(Math.random() * 1000),
+      title: task,
+      completed: false,
+    };
```
[the remark this answers is not in the document]
Comment (PinkMoon25) [reply src/components/TodoContainer.js]: Hi @Owusu-Desmond I have added the UUId package and used for generating unique ids Thanks :+1:

File: README.md [hunk 2]
```diff
@@ -0,0 +1,91 @@
...
+   Check it out live :point_right: [In progress]()
+
+### Prerequisites
+
+have a working git on your localmachine.
+
+## Setup
+
+
+## 🛠 Installation & Set Up
```
[the remark this answers is not in the document]
Comment (PinkMoon25) [reply README.md]: Hi @Owusu-Desmond I have made the changes to readme, Thank you :+1:

**Review by Owusu-Desmond — APPROVED** [review]
Hi @PinkMoon25 

Great job to fulfil all the requirements of this project and the suggestions made in the previous review :+1: 💯 
Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 🎉 

image

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.


------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**